pallets/did/src/migration.rs

```diff
@@ -1,186 +0,0 @@
-// KILT Blockchain – https://botlabs.org
```

Comment:
Not sure we want to delete this. What's your opinion about it @weichweich?

Reply:
Yes this file can go

Reply:
Does it mean we are deleting it or not?

Reply:
Delete. So this can be resolved.